hummingbot/client/config/global_config_map.py

```diff
@@ -330,6 +333,13 @@ def global_token_symbol_on_validated(value: str):
                   validator=lambda v: validate_decimal(v, min_value=Decimal("0"), inclusive=False),
                   required_if=lambda: False,
                   default=Decimal("30")),
+    "tables-format":
+        ConfigVar(key="tables-format",
```

Comment:
For some consistency, I think it would be good to use underscores, `_`, as separators for this `ConfigVar`

Reply:
Good point — addressed.